front/src/applications/operationalStudies/components/SimulationResults/SpeedSpaceChart/SpeedSpaceChart.js

```diff
@@ -296,4 +345,11 @@ SpeedSpaceChart.defaultProps = {
   timePosition: ORSD_GEV_SAMPLE_DATA.timePosition,
   consolidatedSimulation: ORSD_GEV_SAMPLE_DATA.consolidatedSimulation,
   toggleSetting: () => {},
+  onSetSettings: () => {},
+  dispatchUpdateMustRedraw: () => {
+    console.log('dispatchUpdateMustRedraw called');
+  },
+  onSetBaseHeightOfSpeedSpaceChart: ({ ...args }) => {
+    console.log('onOffsetTimeByDragging called');
+  },
```

Comment:
```suggestion
```

Reply:
In that can we need to change our prettier rules. Should be discussed on front chapter

Reply:
Sorry I did not understood well, I will keep a defaut function but remove the logs

Reply:
Done